content/en/api/tracing/code_snippets/send_trace.rb

```diff
@@ -1,11 +1,12 @@
 require 'net/http'
+require 'json'
 
 # Create IDs.
 TRACE_ID = rand(1..1000000)
 SPAN_ID = rand(1..1000000)
 
 # Start a timer.
-START = Time.now.to_f
+START = Time.now.to_i
```

Comment:
I think the most idiomatic way to make the time calculation here would be similar to what we do in our [`Span.rb` class](https://github.com/DataDog/dd-trace-rb/blob/7e0ae6e72945e347deb2827b84907ba60f7d85c2/lib/ddtrace/span.rb#L239-L240).

```ruby
START_TIME = Time.now

sleep 2

END_TIME = Time.now

START = (START_TIME.to_f * 1e9).to_i
DURATION ((END_TIME - START_TIME) * 1e9).to_i
```

Reply:
thanks @marcotc , updated